### DISCUSSION TITLE
Disband the team

### DISCUSSION THREAD

**Discussion by nquinteropato111191** [PaddlePaddle/PaddleSpatial#163] [BaiduKDDCup2022] [answered]
Hi,
I have been invited to a team by team leader, but it was eventually created another team.
Could you please disband this team so that i can be added to the correct one?.
Thanks and regards.
![image](https://user-images.githubusercontent.com/105211288/172888566-6d5a980a-a7dc-4e46-8f6d-fbd97d36aa12.png)

**Comment by begovegar** [comment]
Hi, this would be the team in which  _nquinteropato111191_ should be included:
**Innova Team**, with 4 members at the moment.
I generated this link to invite him:
https://aistudio.baidu.com/studio/match/152/team/invite/382337ee3d5940ccb1fa7937980ef097
Please let us know if you can solve this issue. He tried to disband the team but he wasn't allowed to do it.
Thanks!

**Comment by ramber1836** [comment]
Sorry, since @nquinteropato111191 had submitted results, disbanding a team is not allowed. 
The easiest way is to register for the competition with another email.
Thanks for your understanding.

**Comment by begovegar** [comment]
Hi, I have added a new member to Innova Team, it's @nquinteropatodatascience (previously registered as @nquinteropato111191),
user name: AIStudio2399162
Please confirm this won't be an issue as we have informed you about the mistake. 
Thanks!

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

Our system currently does not support canceling a team.  If you do not use the dismissed team account to submit new models intensively, we will not consider this case disqualified.
 Thanks a lot for your support.

Regards
